src/components/forms/Button.js

```diff
@@ -34,9 +34,12 @@ const Button = ({
 
   if (link) {
     return (
-      <div className={classnames({
-        'rounded-md shadow': color === 'primary' || color === 'secondary',
-      })}>
+      <div
+        className={classnames({
+          'rounded-md shadow': color === 'primary' || color === 'secondary',
+          'inline-block': fullWidth === false,
```

Comment:
I realised that link buttons were always full width even when you didn't want them to be. This change fixes that.